### PR TITLE
Added several commands to Darktrace integration

### PR DESCRIPTION
**Pull request by simmyno**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adding some more commands the the Darktrace integration, seeing as it currently supports a limited amount of API calls. These have been tested in our own integration in a forked integration, and works fine. 

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by yaakovpraisler — CHANGES_REQUESTED** [review]
Looking good. Thanks for the contribution!
Please see my comments -

1. Please update the README file with your new commands.
2. Please bump the version and add release notes - https://xsoar.pan.dev/docs/documentation/release-notes.
3. I also saw that all the other commands have unit tests, so please add tests to the commands you added as well.

**Comment by yaakovpraisler** [comment]
Hi @simmyno - What's the status here?

**Review by simmyno — COMMENTED** [review]
Still in progress, have been a bit busy with other projects. Will update what is needed

**Comment by simmyno** [comment]
@yaakovpraisler Also not very familiar with how Github is setup in pull request, but will figure it out eventually

**Comment by yaakovpraisler** [comment]
@simmyno It's okay. Let me know when you finish.
Also I saw that the last commits were made by another GitHub account, so you'll have to sign in the CLA from that account, as you can see [above](https://github.com/demisto/content/pull/13905#issuecomment-889684831).

File: Packs/Darktrace/Integrations/Darktrace/README.md [hunk 1]
```diff
@@ -251,6 +251,64 @@ Unacknowledges a model breach as specified by Model Breach ID
...
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| cid | Darktrace components ID | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Darktrace.Model | Dictionary | Details of the component |
```
Comment (yaakovpraisler) [inline Packs/Darktrace/Integrations/Darktrace/README.md:310]:
```suggestion
| Darktrace.Component | Dictionary | Details of the component |
```

File: Packs/Darktrace/Integrations/Darktrace/README.md [hunk 2]
```diff
@@ -251,6 +251,64 @@ Unacknowledges a model breach as specified by Model Breach ID
...
+| pbid | Darktrace model breach ID | Required | 
+| endtime | Endtime of data retrieved | Not Required | 
+| count | The amount of lines returned | Not Required | 
+| offset | The offset of data pulled | Not Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Darktrace.ModelBreach.breach_details | Dictionary | Details of the model breach |
```
Comment (yaakovpraisler) [inline Packs/Darktrace/Integrations/Darktrace/README.md:273]:
```suggestion
| Darktrace.ModelBreach| Dictionary | Details of the model breach |
```

File: Packs/Darktrace/Integrations/Darktrace/Darktrace.yml [hunk 3]
```diff
@@ -478,7 +478,45 @@ script:
...
+      defaultValue: "100"
+      description: Specifies the maximum number of items to return. Default 100
+    - name: offset
+      defaultValue: "0"
+      description: Offset of starting point for fetching details of the modelbreach
+    description: Retrieve additional details on a modelbreach
+    outputs:
+    - contextPath: Darktrace.ModelBreach
+      description: ''
+      type: Unknown
```
Comment (yaakovpraisler) [inline Packs/Darktrace/Integrations/Darktrace/Darktrace.yml:498]: Please fill in the description and add the type of the returned value. Also in the following outputs.

File: Packs/Darktrace/Integrations/Darktrace/Darktrace_test.py [hunk 4]
```diff
@@ -361,3 +361,102 @@ def test_get_entity_details(requests_mock):
...
     assert integration_response.outputs == expected_response
     assert integration_response.outputs_prefix == 'Darktrace.EntityDetails'
+
+
+def test_get_modelbreach_details(mocker):
+    """Tests the list-similar-devices command function.
+    Configures requests_mock instance to generate the appropriate
+    get_alert API response, loaded from a local JSON file. Checks
+    the output of the command function with the expected output.
+    """
```
Comment (yaakovpraisler) [inline Packs/Darktrace/Integrations/Darktrace/Darktrace_test.py:371]: Seems this is taken from another test. Please edit these docs (also on the following).

**Review by yaakovpraisler — APPROVED** [review]
Good job with the tests! - I've added a few more individual comments, please fix them and we're done.

**Review by yaakovpraisler — APPROVED** [review]

**Pull request merged**